src/events/message/messageCreate.ts

```diff
@@ -111,7 +87,7 @@ export default class extends Event {
 		}
 	}
 
-	private async safeReply(context: CommandContext, text: string): Promise<Message| undefined> {
+	private async safeReply(context: CommandContext, text: string): Promise<Message | undefined> {
```

Comment:
This change is already in #212, changing it here will cause a merge conflict. Can only avoid it by `git reset --soft` to uncommit the commit but keep the changes staged, removing the change, and then force pushing the new commit. Up to you if you want to deal with that or the conflict.